includes/payment-methods/class-wc-stripe-upe-payment-method-cc.php

```diff
@@ -133,6 +133,12 @@ public function requires_automatic_capture() {
 	 * @return string
 	 */
 	public function get_testing_instructions() {
+		// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace
+		$calling_method = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS, 2 )[1]['function'] ?? '';
```

Comment:
I'm so unhappy with this approach. I hope someone has a suggestion on how to do this differently. I need to check the calling class to avoid a loop call.

Reply:
I found a way to do the same by introducing a new parameter (which I tried to avoid, but it is better than using `debug_backtrace`).